src/content/docs/guides/styling/css-modules.mdx

```diff
@@ -0,0 +1,105 @@
+---
+title: CSS Modules
+draft: false
+---
+
+import CodeSample from "~/components/CodeSample/CodeSample.astro"
+
+# CSS Modules
+
+CSS Modules are CSS files in which class names, animations, and media queries are scoped locally by default. CSS modules let you write your styles in CSS but Solid consumes them as Javascript objects. Once bundled class names referenced from CSS module files are given unique names in order to avoid selector name collisions. Another advantage of using CSS modules over normal CSS files is the fact that only selectors referenced are bundled.
```

Comment:
```suggestion
CSS modules let you write your styles in CSS files and import them in Solid components as an object of classes. 

```css 
// App.module.css

.button {
	border: 1px solid blue;
}

.container {
	display: flex;
}
```

```jsx
// App.jsx
import styles from "./App.module.css"

export default function App() {
	return <div class={styles.container}>
		My button: <button class={styles.button}>Click me!</button>
	</div>
}
```

Once bundled, class names referenced from CSS module files are given unique names in order to avoid selector name collisions.

{/*TODO: OUTPUT EXAMPLE*/}
 
Another advantage of using CSS modules over normal CSS files is the fact that only selectors that are referenced are bundled.
```